deploy/static/02-trivy-operator.rbac.yaml

```diff
@@ -44,23 +44,21 @@ rules:
       - get
       - list
       - watch
+  # Permissions required to read image pull secrets (also through SA)
   - apiGroups:
       - ""
     resources:
       - secrets
       - serviceaccounts
     verbs:
-      - list
-      - watch
       - get
-      - create
-      - update
+  # Permissions required to create scan job image pull secrets
   - apiGroups:
       - ""
     resources:
       - secrets
     verbs:
-      - delete
+      - create
```

Comment:
ordering look weird to me , I prefer see resource and what can de done with it rather the separating by functionality
 ```
- apiGroups:
      - ""
    resources:
      - secrets
    verbs:
      - get
      - create
```

Reply:
Fixed. PTAL